content/en/docs/Getting started/configuration.md

```diff
@@ -0,0 +1,221 @@
+* Shared Secret set to the same as META_SECRET
+* Enable SSL Verification
+* Just The Push Event
+* Check "Active" to turn it on
+
+This "secret" is an arbitrary string that authenticates the push to the Elekto server.  It can be any string, such as one created by a password generator or a passphrase you like.  This string then gets set in META_SECRET. 
+
+## The Environment File
+
+Elekto is designed to accept its runtime configuration as ENV variables in its shell environment.  A sample `.env.example` file can be found in the base directory of the Elekto source.  These ENV configuration variables are not expected to change frequently, or at all, for any particular running Elekto instance.  Changing them generally requires restarting Elekto.
+
+All of these env variables need to be set before starting Elekto as a uwsgi application, or even in developer mode; without them, Elekto will error out and refuse to start. You can set this up however you please:
+
+* as the `.bashrc` for the elekto application user
+* as ENV variables for a container running Elekto
+* preloaded in a systemd unit file
+* injected through a ConfigMap and a Secret into an Kubernetes pod
+
+Since we use ENV for Elekto configuration, this does mean that Elekto must be launched under a shell.
+
+### ENV Variables
+
+What follows are the ENV variables that must be set for Elekto to run.  
+
+#### Application Information
+
+**APP_NAME**
+
+*Required* Name of the Elekto instance you are running.  Displays in some parts of the UI.  For user information only.
+
+Example: `k8s.elections`
+
+**APP_ENV**
+
+*Optional* Status of this Elekto instance, for CI/CD purposes.  Not used internally by Elekto.
+
+Example: `production` or `development` 
+
+**APP_KEY**
+
+*Optional* Seed string for Flask encryption if running Flask in standalone mode.  Not required if fronting with an HTTPS webserver.
+
+Example: ``
+
+FIXME
+
+**APP_DEBUG**
+
+*Required* Whether to run in Debug mode for troubleshooting purposes.  In Debug mode, will output stack traces for errors and reload templates.
+
+Example: `True` or `False`
+
+**APP_URL**
+
+*Optional* URL of the Elekto instance.  Used by some uwsgi and/or Nginx configurations.  Not used internally by Elekto.
+
+Example: `http://elections.knative.dev`
+
+**APP_PORT**
+
+*Optional* Used in some uwsgi start scripts, and when running Flask in standalone mode.  Port on which to serve Elekto.
+
+Example: `5000`
+
+**APP_HOST**
+
+*Optional* Used in some webserver startup scripts, and by the Flask server in standalone mode.  Name of the host served by this Elekto instance.
+
+Example: `localhost`
+
+**APP_CONNECT**
+
+*Optional* Whether to serve uwsgi over HTTP or via a local unix socket.  Used by some startup scripts; see `entrypoint.sh` for an example.  
+
+Example: `http` or `socket`
+
+#### Database Connection
+
+**DB_CONNECTION**
+
+*Required* Which database connection type to use.  Currently only postgresql, mysql, and sqlite are supported.  
```

Comment:
OK, so this would be a problematic change, because the app expects the DB names to be all-lower-case, even if the proper names are mixed case.  So probably code-quoting them instead is going to be the way to go.

Reply:
I hadn't realized. I agree, in that case code-quoting them would be best.

Reply:
Just realized they're code-quoted below, so that's covered in the examples.